src/components/SkippedRequestView.vue

```diff
@@ -118,11 +116,11 @@ watch(
     </div>
     <div
       v-if="requestStore.skippedRequestsForApproval.length > 1"
-      class="border-gray-800 dark:border-gray-300 dark:shadow-md bg-white-300 dark:bg-[#292929] border rounded-md absolute w-[90%] h-full -bottom-2 left-1/2 transform -translate-x-1/2 z-20 shadow shadow-black-600"
+      class="border-gray-800 dark:border-gray-300 bg-[#CCD8DC] dark:bg-black-crayola border rounded-md absolute w-[90%] h-full -bottom-2 left-1/2 transform -translate-x-1/2 z-20"
```

Comment:
Can we add this color to tailwind.config?

Reply:
ok